.module_migration.sh

```diff
@@ -22,6 +22,26 @@ psql postgresql://postgres:test@localhost:5432/postgres -c "CREATE DATABASE __ex
 # Now, we need to move out of the root directory to pull our trickery below
 cd src
 
+# Blow away the existing migration for the specified module, if one exists
+rm -rf modules/${MODULE}/migration
+
+# Get the list of modules this module depends upon and include itself for use in the temporary
+# TypeORM configuration
+MODULES=`cat modules/${MODULE}/module.json | jq -r ".dependencies[.dependencies | length] |= \"${MODULE}\" | .dependencies | join(\":\")"`
+readarray -d ":" -t MODARR <<< "${MODULES}"
+
+# Convert the array of modules into configuration paths for TypeORM
+ENTITIES=""
+MIGRATIONS=""
+NL=$'\n'
+for (( n=0; n < ${#MODARR[*]}; n++))
+do
+  MOD=`echo ${MODARR[n]} | xargs`
+  ENTITIES="${ENTITIES}\"modules/${MOD}/entity/*.ts\",${NL}"
+  MIGRATIONS="${MIGRATIONS}\"modules/${MOD}/migration/*.ts\",${NL}"
+done
+
+# Generate the TypeORM config
 cat <<EOF > ormconfig.js
```

Comment:
nit: you could set env variables if you wanted which feels more natural for a bash script https://typeorm.io/#/using-ormconfig/using-environment-variables